dubbo-metadata/dubbo-metadata-report-redis/src/test/java/org/apache/dubbo/metadata/store/redis/RedisMetadataReportTest.java

```diff
@@ -60,15 +60,15 @@ public void constructor(TestInfo testInfo) throws IOException {
         String methodName = testInfo.getTestMethod().get().getName();
```

Comment:
Below this line you would put
    ExecutableProvider provider = () -> new File(System.getProperty("my.custom.property"));